### ISSUE TITLE
using System.Drawing; was not added as one of the auto generated using statements

### ISSUE THREAD

**Issue by AndreaWilliamsIteris** [RandomEngy/UnitTestBoilerplateGenerator#30]
### Installed product versions
- Visual Studio: [example 2017 Professional]
- This extension: [example 2.7.0]

### Description
When I created boiler plate unit test, the using statement did not get added and I had to add in myself.
using System.Drawing;

### Steps to recreate
1. Add this to code
			var newLabel = new DevExpress.XtraEditors.LabelControl
			{
				Text = name,
				Width = 150,
				Font = font,
				AutoSizeMode = LabelAutoSizeMode.Vertical,
				AutoSize = true,
				Dock = DockStyle.Left,
			};

2. Create a Unit test
3. Font has red squiggle because using statement is missing.

### Current behavior
using statement is missing because system.drawing is not referenced

### Expected behavior
using System.Drawing; is included in the unit test file and Font doesn't have a missing reference.

**Comment by RandomEngy** [comment]
Did you put System.Drawing in the `Extra "using" namespaces` field in Options? 
![image](https://user-images.githubusercontent.com/9450861/178093192-199e568b-3120-488d-82fc-adee3e3129a3.png)

**Comment by RandomEngy** [comment]
Maybe? Many times, you don't actually want all of the using statements that the source file has. I'll consider it as an option.

**Comment by AndreaWilliamsIteris** [comment]
But you can always remove them later.  I'd rather have too manyand delete the extra's than have to hunt down what is missing, personally.

**Comment by RandomEngy** [comment]
Yeah that field is really more for if you included some extra test library that was always used in the template. It's a reasonable feature request. Might be a while until I can get to it as I'm busy with some other projects, but I'm always accepting PRs.
